src/pages/search.vue

```diff
@@ -145,10 +136,23 @@ export default defineComponent({
         !isShallowEqualObjects(newRoute.query, oldRoute.query)
       ) {
         const { query, path } = newRoute
-        this.setSearchStateFromUrl({ urlQuery: query, path })
+        const searchStore = useSearchStore(this.$nuxt.$pinia)
+        searchStore.setSearchStateFromUrl({ urlQuery: query, path })
         await this.fetchMedia()
       }
     },
   },
+  methods: {
+    handleTab(event: KeyboardEvent, element: string) {
+      if (this.showScrollButton.value && element !== 'scroll-button') {
+        return
+      }
+      focusIn(document.getElementById('__layout'), Focus.First)
+    },
+    fetchMedia(...args: unknown[]) {
+      const mediaStore = useMediaStore(this.$pinia)
+      return mediaStore.fetchMedia(...args)
+    },
```

Comment:
Because you can't return functions in `setup` and also use Options API for some things or else Nuxt will raise the "cannot stringify unbound function" error in the console. See https://github.com/WordPress/openverse-frontend/issues/1237 for where we've fixed this before.